### ISSUE TITLE
About training

### ISSUE THREAD

**Issue by zagainov-sd** [Hope1337/YOWOv3#37]
Hi, I wanted to train YOWO to detect only one action and faсed the problem that YOWO makes false positives detections. How can I add an examples of "no action" to training dataset (As I understand unannotated train data lead to loss = nan)? Or may be there are another solutions.

**Comment by Hope1337** [comment]
@zagainov-sd Yes, you can do that. Actually, you need a huge dataset for generalization of model reach the desire point.

**Comment by zagainov-sd** [comment]
Thank you for your responce. But I still can't understand how to add unannotated data (it will lead to loss = nan as I understand). Now I have dataset with one class. And I want to add background class with no annotation. 
Thank you!

**Comment by Hope1337** [comment]
@zagainov-sd it requires some modification in loss function which currently is not supported, that mean you have to carry out by yourself (i.e bounding box with background label will be computed as a loss component?)

**Comment by zagainov-sd** [comment]
Now it's clear, hard task) Do you plan to add this functionality? What do you think about this idea: if the background class does not have an annotation YOWO will understand that there should be no predictions. So it will allow to increase dataset without annotating. Is it possible?
